docker-compose.yml

```diff
@@ -53,6 +53,12 @@ services:
     environment:
       SERVICES: secretsmanager
       DEBUG: 1
+  registry:
+    image: registry:2
+    ports:
+      -  "5000:5000"
```

Comment:
I don't know if it will affect anything at the moment, but FYI the kube tests also start a registry on port 5000.

Reply:
moved to 5001